taraxa/state/dpos/precompiled/api.go

```diff
@@ -27,13 +28,21 @@ type Config = struct {
 	CommissionChangeFrequency   types.BlockNum
 	DepositDelay                types.BlockNum
 	WithdrawalDelay             types.BlockNum
-	GenesisState                []GenesisStateEntry
```

Comment:
This struct should be deleted if we are not using it anymore... That 1 usage in test must be changed to the GenesisValidator